masterfermin02/coronavirusrd
# Added service to handle the data from the API

This pr is to add the services that will have the responsibility of reading the information about corovirus from rapidapi.

## QA note

When you go to the home page of the dashboard you should see all the information updated about the coro virus in DR
- `Cases`
- `Observes`
- `Deaths`
- `Recovereds`

## DEV note 
Should run `npm install` to install some dependencies necesary on this pr
